app/assets/stylesheets/catarse_bootstrap/_extra.sass

```diff
@@ -27,6 +27,9 @@ select.date
 .deliver-at-date
   width: 230px
 
+.project_preview_tab
+  float: right
```

Comment:
Our tab js is not working properly with tabs in separate rows, so I had to put it in the same row and just change the style.